pkgs/tools/misc/psw/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv
```

See https://github.com/NixOS/nixpkgs/issues/108938.

```suggestion
{ lib
```

---

Fixed.